.github/workflows/build.yml

```diff
@@ -13,6 +13,7 @@ on:
       - beta
       - rc
       - feature/local-dev-dx
+      - feature/comprehensive-updates
```

Comment:
I recommend that we audit this list of branches and patterns.  Doesn't seem like we should be listing specific feature branches here, but should have a pattern like `feature/*`.   If we don't want to trigger the build on every feature branch, then come up with a naming convention that is used to trigger the build.

Reply:
True, these two are only here to facilitate development. I'll make sure to clear them before any merging.